daprdocs/content/en/operations/resiliency/configure-policies.md

```diff
@@ -0,0 +1,282 @@
+kind: Resiliency
+metadata:
+  name: resiliency
+scopes:
+  # optionally scope the policy to specific apps
+spec:
+  policies:
+    timeouts:
+      # timeout policy definitions
+
+    retries:
+      # retry policy definitions
+
+    circuitBreakers:
+      # circuit breaker policy definitions
+
+  targets:
+    apps:
+      # apps and their applied policies here
+
+    actors:
+      # actor types and their applied policies here
+
+    components:
+      # components and their applied policies here
+```
+
+### Policies
+
+#### Timeouts
+
+Timeouts can be used to early-terminate long-running operations. If a timeout is exceeded the operation in progress will be terminated if possible and an error is returned. Valid values are of the form `15s`, `2m`, `1h30m`, etc
+
+Example definitions:
+```yaml
+spec:
+  policies:
+    # Timeouts are simple named durations.
+    timeouts:
+      general: 5s
+      important: 60s
+      largeResponse: 10s
+```
+
+#### Retries
+
+Retries allow defining of a retry stragegy for failed operations. Requests failed due to triggering a defined timeout or circuit breaker policy will also be retried per the retry strategy. The following retry options are configurable:
+
+- `policy`: determines the backoff and retry interval strategy. Valid values are `constant` and `exponential`. Defaults to `constant`.
+- `duration`: determines the time interval between retries. Default: `5s`. Only applies to the `constant` `policy`. Valid values are of the form `200ms`, `15s`, `2m`, etc
+- `maxInterval`: determines the largest interval between retries to which the `exponential` backoff `policy` can grow. Additional retries will always occur after a duration of `maxInterval`. Defaults to `60s`. Valid values are of the form `5s`, `1m`, `1m30s`, etc
+- `maxRetries`: The number of retries to attempt. `-1` denotes an indefinite number of retries. Defaults to `-1`.
+
+The exponential backoff window uses the following formula:
+```
+BackOffDuration = PreviousBackOffDuration * (Random value from 0.5 to 1.5) * 1.5
+if BackOffDuration > maxInterval {
+  BackoffDuration = maxInterval
+}
+```
+
+Example definitions:
+```yaml
+spec:
+  policies:
+    # Retries are named templates for retry configurations and are instantiated for life of the operation.
+    retries:
+      pubsubRetry:
+        policy: constant
+        duration: 5s
+        maxRetries: 10
+
+      retryForever:
+        policy: exponential
+        maxInterval: 15s
+        maxRetries: -1 # Retry indefinitely
+```
+
+##### Circuit Breakers
+
+???
+
+Example:
+
+```yaml
+spec:
+  policies:
+    circuitBreakers:
+      pubsubCB:
+        maxRequests: 1
+        interval: 8s
+        timeout: 45s
+        trip: consecutiveFailures > 8
+```
+
+### Targets
+
+#### Apps
+
+Allows applying of `retry`, `timeout` and `circuitbreaker` policies to service invocation calls to other Dapr apps. Policy assignments are optional.
+
+Example
+```yaml
+specs:
+  targets:
+    apps:
+      appB:
```

Comment:
We need to mention that the value here is a Dapr App-ID and it's the policy applied to that service when it's the target ("callee")